solr/solr-ref-guide/src/nested-documents.adoc

```diff
@@ -36,17 +36,20 @@ Nested documents may be indexed via either the XML or JSON data syntax, and is a
 
 === Schema Notes
 
- * The schema must include indexed fields field `\_root_`. The value of that field is populated automatically and is the same for all documents in the block, regardless of the inheritance depth.
- Fields `\_nest_path_`, `\_nest_parent_` can be configured to store the path of the document in the hierarchy, and the unique `id` of the parent in the previous level.
- These 2 fields will be used by NestedUpdateProcessor URP, which is implicitly configured under Solr 8, when `\_root_` field is defined.
+ * The schema must include indexed field `\_root_`. The value of that field is populated automatically and is the same for all documents in the block, regardless of the inheritance depth. The id of the top document in every nested hierarchy is populated in this field.
+ * `\_nest_path_` can be configured to store the path of the document in the hierarchy
+ * `\_nest_parent_` can be configured to store the `id` of the parent in the previous level
  * Nested documents are very much documents in their own right even if certain nested documents hold different information from the parent.
    Therefore:
  ** the schema must be able to represent the fields of any document
  ** it may be infeasible to use `required`
  ** even child documents need a unique `id`
 
 
-=== Legacy Schema Notes
+=== Rudimentary Root-only schemas
+ * These schemas do not contain any other nested related fields apart for `\_root_`. +
+   In this mode relationship types(field names) between parents and their children are not saved.
+   All children are indexed under the `\_childDocuments_` field.
```

Comment:
I think this sentence is misleading.  You can choose to provide the data to Solr either way in fact, and regardless we do not "index" children by field.  I think you were trying to say that the child doc transformer will return the list flat and unlabelled.

Reply:
Oops.
My bad.
I'll change it to returned.